src/main/java/ai/datahunters/md/server/analytics/AnalyticsHandler.java

```diff
@@ -0,0 +1,25 @@
+package ai.datahunters.md.server.analytics;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.ExecutionException;
+
+@AllArgsConstructor
+@Component
+public class AnalyticsHandler {
+
+    private AnalyticsService service;
+
+    public ResponseEntity getAnalytics()  {
+        try {
+            return ResponseEntity.ok(new AnalyticsData(service.photoCount()));
```

Comment:
you dont need that Response Entity. You can just return AnalyticsData from here.